projects/plugins/jetpack/extensions/shared/use-is-user-connected.ts

```diff
@@ -0,0 +1,10 @@
+import { useConnection } from '@automattic/jetpack-connection';
+import { isSimpleSite } from '@automattic/jetpack-shared-extension-utils';
+
+const useIsUserConnected: () => boolean = () => {
```

Comment:
Someone who knows more about this stuff than I do might want to comment on this versus the `isCurrentUserConnected()` function from `@automattic/jetpack-shared-extension-utils`, and also whether the AI Assistant block's `isUserConnected` from `projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/connection/index.ts` should be combined with this somehow.

But as a quick fix to unblock Simple deploys, let's do this an iterate IMO.

Reply:
Ah I didn't realize the `isCurrentUserConnected` function was in a package like that, my mistake